src/main/java/seedu/address/logic/parser/ApplicationBookParser.java

```diff
@@ -35,7 +37,7 @@ public class ApplicationBookParser {
     public Command parseCommand(String userInput) throws ParseException {
         final Matcher matcher = BASIC_COMMAND_FORMAT.matcher(userInput.trim());
         if (!matcher.matches()) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE));
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT));
```

Comment:
Should put this back as part of the invalid command error message

```suggestion
            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE));
```

Reply:
Thanks for catching that!